pkg/dataplane/baremetal.go

```diff
@@ -89,6 +89,7 @@ func DeployBaremetalSet(
 						return fmt.Errorf("%s gateway is missing", dataplanev1.CtlPlaneNetwork)
 					}
 					baremetalSet.Spec.CtlplaneGateway = *res.Gateway
+					baremetalSet.Spec.CtlplaneNetmask = string(ipNet.Mask)
```

Comment:
I think `ctlplaneNetmask` is not used at all atm. ctlplaneIP is in CIDR format[1] and that's used instead in openstack-baremetal-operator. 

[1] https://github.com/openstack-k8s-operators/openstack-operator/blob/main/pkg/dataplane/baremetal.go#L87

Reply:
Ok, let's remove it from baremetal-operator then, and that should solve this problem that I was trying to address. So we can just close this PR.